gateway-ha/src/main/java/io/trino/gateway/ha/config/HaGatewayConfiguration.java

```diff
@@ -20,6 +20,7 @@
 
 public class HaGatewayConfiguration
 {
+    private Map<String, String> airliftConfig = new HashMap<>();
```

Comment:
`airliftConfig` may not be intuitive to users unfamiliar with the implementation details. If we set `node.environment` in the code, then something like `httpConfig` would be more obvious